.github/workflows/github-build-release.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+' # Push events to matching semver, i.e. 10.0.0
+
+name: Build and create release
+
+jobs:
+  build:
+    name: Build release binary
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build
+        run: mvn install
```

Comment:
If we are just building for release we should do a package here, no need to do a full install command

Reply:
Shall update